tests/schemas/test_census_household.json

```diff
@@ -0,0 +1,8148 @@
+{
```

Comment:
I don't think this should be included in this repo, it will quickly become outdated.